v2.0/architecture/transaction-layer.md

```diff
@@ -80,9 +80,9 @@ For more detail about the risks that large clock offsets can cause, see [What ha
 
 ### Timestamp Cache
 
-To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a Timestamp Cache, which shows the high-water mark for values being read.
+To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a timestamp cache, which shows the high-water mark for values being read.
 
-Whenever a write occurs, its timestamp is checked against the Timestamp Cache. If the timestamp is less than the Timestamp Cache's latest value, we attempt to move the timestamp for its transaction forward to a later time. In the case of serializable transactions, this causes them to restart in the second phase of the transaction.
+Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to push the timestamp for its transaction forward to a later time. In the case of serializable transactions, this might cause them to restart in the second phase of the transaction (see [read refreshing](#read-refreshing)).
```

Comment:
Did you get feedback about the "causes" vs. "might cause" technical detail?

Reply:
Nope! I just went with the one that looked slightly "righter".

Looking at `git blame` it looks I got lucky and @andreimatei authored the replacement text in 73fa655